10up/brightcove-video-connect
# Clear filtered results when input search is empty

### Description of the Change

This change updates to clear filtered results when the input search is empty.

### Alternate Designs

### Benefits

There is no need to click on the Search button to reset the results.

### Possible Drawbacks

### Verification Process

1. Add the Brightcove block and edit it
2. Type in the search field and hit Enter to filter the results
3. Erase the search term and check if the results are reset

### Checklist:






### Applicable Issues

### Changelog Entry
